docs/README.md

```diff
@@ -0,0 +1,49 @@
+Initial Setup and Upgrading
+===========================
+
+Getting Started
+---------------
+- [ ] Use composer to create a new project, starting from this repository as a template:
+    ```bash
+    composer create-project lastcall/drupal-scaffold PROJECTNAME
+    ```
+- [ ] Bring up the Docker containers and enter the Drupal container.
+    ```bash
+    docker-compose up -d drupal varnish
+    docker-compose exec drupal /bin/bash
+    ```
+- [ ] From the repository root (`/var/www` inside the Drupal container), install NPM and composer dependencies:
+    ```bash
+    yarn install
+    composer install
+    ```
+- [ ] Visit the Drupal site in your browser to install Drupal and continue.  The default URL will be `http://localhost:8080`.
```

Comment:
What if we just reverse the ports?  I agree straight Drupal access is probably what you want for local development, so how about if our instructions leave out anything about the varnish container, and we just expose Drupal on 8080?  I guess it's technically a BC break, but this is 2.0 baby!

Reply:
Haha sure that sounds good! I like having access directly to the Drupal container be the default.

We could always add a docs page for varnish that explains how to access it with the cache. Don't think it needs to happen for the initial merge though.

Kind of a tangent from the initial comment, but since it's 2.0 maybe now's the time to think about it - does it make sense to keep the Varnish container enabled by default? It's not really something that I've ever intentionally used locally, not sure about others who are using it.  Maybe it's something that starts commented out in `docker-compose.yml` (like solr), so it can easily be added if someone wants it... just a thought, I'd be fine with just switching the ports too.

Reply:
Yeah, that's a good point.  Less enabled stuff is better.  Let's do that as a followup to this PR.  I want to get this merged so we can review things in smaller chunks. I wonder if this works?

- [ ] Create followup for disabling Varnish container by default.